admin-dev/themes/new-theme/js/pages/order/view/order-documents-refresher.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+import Router from '@components/router';
+import OrderViewPageMap from '@pages/order/OrderViewPageMap';
+import InvoiceNoteManager from '../invoice-note-manager';
+
+const $ = window.$;
+
+export default class OrderDocumentsRefresher {
+  constructor() {
+    this.router = new Router();
+    this.invoiceNoteManager = new InvoiceNoteManager();
+  }
+
+  refresh(orderId) {
+    $.ajax(this.router.generate('admin_orders_get_documents', {orderId}))
+      .then((response) => {
+        $(OrderViewPageMap.orderDocumentsTabCount).text(response.total);
+        $(OrderViewPageMap.orderDocumentsTabContentTable).html(response.html);
+        this.invoiceNoteManager.setupListeners();
+      });
```

Comment:
No error handling necessary?

Reply:
@NeOMakinG I don't think so, there is no "action", it's only fetching data to refresh a part of the page. And even if it fails, if you refresh the page you'll be up-to-date. But maybe I'm wrong. WDYT?

Reply:
But what tell you to refresh the page if you see nothing telling you ? If thats the normal behavior/workflow, let's doing this, or maybe just create an alert, @PrestaShop/prestashop-core-developers what's the normal workflow for error handling on JS side of PS?

Reply:
@NeOMakinG If you don't see your new invoice in the document tab, you'll may think to reload the page?
Anyway, if we decide to handle errors here, we should add error handling in the [price refresher](https://github.com/PrestaShop/PrestaShop/blob/1.7.7.x/admin-dev/themes/new-theme/js/pages/order/view/order-prices-refresher.js#L38) as well

Reply:
That would be a bug tbh, even if peoples refresh, they'll want to create an issue to fix that

Yes of course, we need to handle errors everywhere :/

But as everyone looks to approve the PR, maybe that's something to do in another issue/pr